docs/developer/custom-fields.rst

```diff
@@ -16,18 +16,22 @@ that can be used in Django models.
 A model field which provides a random key or token, widely used across
 openwisp modules.
 
+.. important::
+
+    If the value of the **fallback field** is equal to the fallback value,
+    then the field will store ``None`` in the database.
+
 ``openwisp_utils.fields.FallbackBooleanChoiceField``
 ----------------------------------------------------
 
 This field extends Django's `BooleanField
 <https://docs.djangoproject.com/en/4.2/ref/models/fields/#booleanfield>`_
 and provides additional functionality for handling choices with a fallback
-value. The field will use the **fallback value** whenever the field is set
-to ``None``.
+value. The field will return the **fallback value** whenever the field is
+set to ``None``.
```

Comment:
```suggestion
value.

.. note::

    - The field will return the **fallback value** whenever is
      set to ``None``.
    - Setting the same value as the **fallback value** will
      save ``None`` (NULL) in the database.
```

Please make a partial for this note and include it in all the other occurrences.